stdlib/2and3/distutils/cmd.pyi

```diff
@@ -13,3 +13,28 @@ class Command:
     def finalize_options(self) -> None: ...
     @abstractmethod
     def run(self) -> None: ...
+
+    def announce(self, msg: str, level: int = ...) -> None: ...
```

Comment:
The argument types should probably be Text virtually always; Python 2 users should be able to pass in unicodes.

Reply:
(So, to be a little more clear, argument types can and should often be Text instead of str, but for return types and a few other contexts, such as arguments to invariant containers like List, it's more complicated.)

Reply:
Right; I think the disconnect is possibly since stubs are python3-style, but are relevant for python2.